tests/test_source_map.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+from typing import Iterator
+
+import pytest
+
+from ethpm_types.contract_type import SourceMap, SourceMapItem
+
+SOURCE_MAP_FILES = {p.stem: p for p in sorted(Path("tests/data").glob("*.srcmap"))}
+
+
+def serialize(sourcemap: Iterator[SourceMapItem]) -> str:
```

Comment:
So this source code mapping is made-up or is this like a real encoding?

Reply:
If this is a legit encoding, should we make like a `SourceMapItemCollection` class or something?

Reply:
Yes, it is a real encoding (although it is kind of adhoc and made-up by the Solidity team lol)

Why would it need a class?